lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -159,7 +163,7 @@ def sample_from_image(self, num_pixels, reduce=np.sum):
                     max(int(c) - num_pixels, 0) : min(int(c) + num_pixels + 1, y_size), int(t)
                 ]
             )
-            for t, c in zip(self.time_idx, self.coordinate_idx)
+            for t, c in zip(list(self.time_idx), list(self.coordinate_idx))
```

Comment:
Why is `list()` needed here? It probably shouldn't be.

Reply:
It's not. Removed.